src/main/java/de/qabel/desktop/ui/about/aboutPopup/AboutPopupController.java

```diff
@@ -40,4 +156,14 @@ public void hidePopup() {
     public void showPopup() {
         aboutPopup.setVisible(true);
     }
+
+    private void setStyleTextArea(){
+        textAreaPopup.getStyleClass().clear();
+        textAreaPopup.getStyleClass().add("text-area-popup");
+    }
+
+    private void setStyleTextAreaCenter(){
```

Comment:
please take care of the code style (missing space in front of the `{`). Your IDE can ensure the code style for you